application/controllers/handler.js

```diff
@@ -422,30 +422,43 @@ module.exports = {
     const query = {
       username: {
         $regex: username
+      },
+      deactivated:{
+        $not: {
+          $eq: true
+        }
       }
     };
     console.log('query:', query);
 
     return userCtrl.find(query)
-      .then((cursor1) => cursor1.project({username: 1, _id: 1, picture: 1}))
+      .then((cursor1) => cursor1.project({username: 1, _id: 1, picture: 1, country: 1, organization: 1}))
       .then((cursor2) => cursor2.limit(10))
       .then((cursor3) => cursor3.toArray())
       .then((array) => {
         // console.log('handler: checkUsername: similar usernames', array);
         let data = array.reduce((prev, curr) => {
+          let description = curr.username;
+          if (curr.organization)
+            description = description + ', ' + curr.organization;
+          if (curr.country)
+            description = description + ', ' + curr.country;
```

Comment:
This looks a little a strange, but I haven't tested the route yet. Is there a reason you're constructing a description?